clients/java/src/main/java/io/camunda/zeebe/client/impl/ZeebeClientBuilderImpl.java

```diff
@@ -40,26 +40,30 @@
 import java.time.Duration;
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 import java.util.Properties;
 import java.util.concurrent.ScheduledExecutorService;
 
 public final class ZeebeClientBuilderImpl implements ZeebeClientBuilder, ZeebeClientConfiguration {
+
   public static final String PLAINTEXT_CONNECTION_VAR = "ZEEBE_INSECURE_CONNECTION";
   public static final String CA_CERTIFICATE_VAR = "ZEEBE_CA_CERTIFICATE_PATH";
   public static final String KEEP_ALIVE_VAR = "ZEEBE_KEEP_ALIVE";
   public static final String OVERRIDE_AUTHORITY_VAR = "ZEEBE_OVERRIDE_AUTHORITY";
-
   public static final String ZEEBE_CLIENT_WORKER_STREAM_ENABLED =
       "ZEEBE_CLIENT_WORKER_STREAM_ENABLED";
   public static final String DEFAULT_GATEWAY_ADDRESS = "0.0.0.0:26500";
   public static final String DEFAULT_TENANT_ID_VAR = "ZEEBE_DEFAULT_TENANT_ID";
-
+  public static final String DEFAULT_JOB_WORKER_TENANT_IDS_VAR =
+      "ZEEBE_DEFAULT_JOB_WORKER_TENANT_IDS";
+  private static final String LIST_SEPARATOR = ", ";
```

Comment:
❓ Why is the separator `, ` (comma followed by space)? Why not just `,` (comma)?

🔧 Is this a default list separator that can be used for other lists, or is it intended only for tenant ids? If it's the latter we should include `tenantIds` in the name. Otherwise, the name is fine.

Reply:
I added a "comma with a space" separator, as a comma (`,`) is still an allowed character for a tenant ID. I don't expect that a lot of our users will use it, but we should account for it. @korthout do you think it would be better if we used just a comma?

> Is this a default list separator that can be used for other lists, or is it intended only for tenant ids?
There are no default list separators for property files. It's up to us to decide.

:+1: I'll include `tenantIds` in the name for clarity.

Reply:
I think this is a bit confusing to use. I imagine users find it unexpected that it requires comma with space as spaces have meaning in shells. 

```
$ export ZEEBE_DEFAULT_JOB_WORKER_TENANT_IDS=tenant-a, tenantb

$ export | grep tenant
ZEEBE_DEFAULT_JOB_WORKER_TENANT_IDS=tenant-a,
tenantb=''
```

As a workaround, I'll need to use double quotes:

```sh
$ export ZEEBE_DEFAULT_JOB_WORKER_TENANT_IDS="tenant-a, tenantb"

$ export | grep tenant
ZEEBE_DEFAULT_JOB_WORKER_TENANT_IDS='tenant-a, tenantb'
```

We would not have this problem with a comma delimiter.

❓ Would it be possible still to disallow comma in tenantIds?

Reply:
Actually, the regex that we're using (`^[\w\.-]{1,31}$`) excludes commas already :smile: 

I'll just use a `,` (comma) delimiter. :+1: